Detectors/AOD/src/AODProducerWorkflowSpec.cxx

```diff
@@ -793,10 +798,31 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
                      collisionTimeMask);
     auto& trackRef = primVer2TRefs[collisionID];
     // passing interaction time in [ps]
-    fillTrackTablesPerCollision(collisionID, tsTimeStamp * 1E3, trackRef, primVerGIs, recoData, tracksCursor, tracksCovCursor, tracksExtraCursor, mftTracksCursor);
+    fillTrackTablesPerCollision(collisionID, tsTimeStamp * 1E3, trackRef, primVerGIs, recoData,
+                                tracksCursor, tracksCovCursor, tracksExtraCursor, mftTracksCursor);
     collisionID++;
   }
 
+  // filling v0s table
+  for (auto& svertex : secVertices) {
+    auto trPosID = svertex.getProngID(0);
+    auto trNegID = svertex.getProngID(1);
+    int posTableIdx = -1;
+    int negTableIdx = -1;
+    LOG(DEBUG) << "V0s: " << trPosID.getIndex() << " " << trNegID.getIndex();
+    auto item = mV0sIndices.find(trPosID.getIndex());
```

Comment:
Same as above, the `trPosID.getIndex()` is not enough to define the prong track reference, e.g. the it could be ITS-TPC or ITS-TPC-TRD track with the same `getIndex()` but there are completely different track entries.
Use GlobalTrackID as the `mV0sIndices` key and, respectively, mV0sIndices.find(...) should be called with e.g. `trPosID` rather than `trPosID.getIndex()`

Reply:
Thank you for advice @shahor02. I didn't notice that GlobalTrackIndex has a defined hash function, so I tried using `getIndex()`. I have switched to GIndex for keys